src/Donors/Models/Donor.php

```diff
@@ -55,6 +56,7 @@ class Donor extends Model implements ModelCrud, ModelHasFactory
         'phone' => 'string',
         'prefix' => 'string',
         'additionalEmails' => ['array', []],
+        'additionalPhones' => ['array', []],
```

Comment:
Ah, I see your attempting to add the `additionalPhones` just like `additionalEmails` works.  Honestly, I would like to prevent this for now.  The `additionalEmails` key is already problematic and should have been accomplished differently than how it works now.  What we have is really a workaround to compensate for the legacy storage of additional emails.

For this scope, we should only be supporting one phone number for a donor.  So let's remove the additional phone numbers for now and can circle back to a better approach if/when the request for that comes up.

Reply:
Resolved in f214cb7890523c071038f559a9a5eb48901424ba